doc/DeveloperGuide.md

```diff
@@ -94,6 +94,10 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. The system should be able to run in 32-bit and 64-bit systems.
```

Comment:
This is mentioned on the GitHUb page. Think about other NFRs that might be relevant here.